IS7024/Pages/LudlowGarage.cshtml

```diff
@@ -4,74 +4,60 @@
     ViewData["Title"] = "LudlowGarage";
     var events = (List<QuickType.Event>)ViewData["events"];
     var sun = ViewData["sun"];
+    string eventlocation = (string)ViewData["eventlocation"];
```

Comment:
This was added so that the city and state of the venue is pulled from the JSON data for that venue instead of written in.